spec/services/solidus_bolt/accounts/detail_service_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::Accounts::DetailService, :vcr, :bolt_configuration do
+  subject(:api) { described_class.new(access_token: access_token) }
+
+  let(:access_token) { 'Bolt Access Token' }
+
+  describe '#call', vcr: true do
+    it 'receives the correct response' do
+      expect{ api.call }.to raise_error(SolidusBolt::ServerError, 'This action is forbidden.')
+    end
```

Comment:
IMO This failing test is not enough.

Considering we'll add other services which will use AccessToken, we should provide a way to specify it to the local env.

Reply:
Modified this test here
[5a6db27](https://github.com/nebulab/solidus_bolt/commit/5a6db27026af90bed6d07db9857e40f0baca3c19)